src/ifaddrs_posix.rs

```diff
@@ -25,6 +25,7 @@ pub fn do_broadcast(ifaddr: &ifaddrs) -> Option<IpAddr> {
     target_os = "ios",
     target_os = "macos",
     target_os = "openbsd"
```

Comment:
I think this is just missing a comma at the end of the line :)

Reply:
So embarrassing. Commas are my nemesis.

Reply:
Force pushed changes to my branch.